subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -62,6 +62,17 @@ struct ticker_node {
 					  * remainder
 					  */
 #if !defined(CONFIG_BT_TICKER_COMPATIBILITY_MODE)
+#if  defined(CONFIG_BT_TICKER_EXT)
+	u32_t ticks_slot_window;	 /* Window in which the slot
```

Comment:
Can we get these 9 bytes (12 bytes unpacked) of memory supplied by user of the extension feature? Just have a pointer here to reference the memory defined by user/caller of ticker_start_ext interface.

This way only the advertiser will use 16 bytes extra, where as other states/roles use 4 bytes extra.

Reply:
Good idea